### DISCUSSION TITLE
Incognito

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#719] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Incognito no longer opens in an Incognito tab, just a normal one.

**Comment by AshtonDavies** [comment]
Yes, it was patched with LTMEAT in 115.

**Reply by S-PScripts** [reply]
then he should put the words patched after it?

**Reply by AshtonDavies** [reply]
A lot of stuff in here doesn't have patch notes. You need to test things and see what works if you don't know what's patched.

**Reply by AshtonDavies** [reply]
One thing that has a pretty good chance of still working is incognito on the sign-in screen. Haven't tried it on a regular Chromebook and I don't know how Google would go about preventing that on managed devices, because you kinda need to sign into your network to sign into your account for most cases. They seem to patch things in the laziest unexpected ways instead of programically fixing the issues causing the exploits, they often just remove almost the entirety of it from working.

**Reply by TheAnimemer** [reply]
@AshtonDavies I havent heard of opening Incognito on the sign-in screen. Could you tell me the steps/where to find the steps to do it? I'll test it on my Chromebook.

**Reply by AshtonDavies** [reply]
> @AshtonDavies I havent heard of opening Incognito on the sign-in screen. Could you tell me the steps/where to find the steps to do it? I'll test it on my Chromebook.

Okay, technically not the incognito part of it, but the previous DNS address for it used to be the LTBEEF DNS until the incognito exploit was discovered, and my hotspot saved network was left set to it and the captive portal window worked on the sign-in screen with the webview link. That's what I mean if you just want a built-in webview bypass. Might still work idk, haven't cared to try.

**Reply by Mazedotexe** [reply]
Wait, doesn't that imply that it works on 114? Its patched for me and I have it "CAUBed" on 114

**Reply by AshtonDavies** [reply]
> Wait, doesn't that imply that it works on 114? Its patched for me and I have it "CAUBed" on 114

It might actually work below 115. Technically I think the sign in screen captive portal window has been a thing for quite a while now that I think about it.

**Reply by Mazedotexe** [reply]
@AshtonDavies Why doesn't it work for me? I'm on 114... it used to work for a bit and then it would just forget about the sign-in page and say internet was working fine even though it wasnt. Now it just doesn't work.

Edit: nevermind, it works again. The DNS was down. However, like I said, it only lasts for a little bit before closing webview and pretending internet is fine, and then if i try to log in, it throws an error 400 at me.  😐

**Reply by AshtonDavies** [reply]
The Incognito exploit was patched in v115, so it still works in your current version. And yes, the downside to using the captive portal webview is that it closes itself frequently without warning if it notices your connection is working fine.
